neo/Network/P2P/Payloads/GetFullBlocksPayload.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class GetFullBlocksPayload : ISerializable
+    {
+        public const ushort MaxBlocksCount = 500;
+        public uint IndexStart;
+        public short Count;
+
+        public int Size => sizeof(uint) + sizeof(short);
+
+        public static GetFullBlocksPayload Create(uint index_start, short count = -1)
```

Comment:
Can't be `-1` I think.

Reply:
it's possible because it is `public short Count`, unless we change it according to https://github.com/neo-project/neo/pull/1138#discussion_r342511473

Reply:
https://github.com/neo-project/neo/blob/a8486305d8a24cef3f55bd36e6268f082b41bfd1/neo/Network/P2P/Payloads/GetFullBlocksPayload.cs#L28

You have limited it to positive value.

Reply:
Yes I just noticed while processing some of your feedback. I believe what I just commited is what we want. Which is `GetFullBlocksPayload.Create(start_index)` without a count to provide the maximum allowed